utils/date.ts

```diff
@@ -1,11 +1,18 @@
+import { format, parseISO } from "date-fns";
 import { getFormatedDate } from "react-native-modern-datepicker";
 
 export const MINIMUM_DATE = new Date(1900, 1, 1);
 
 export const formatDate = (date: Date) => {
   return getFormatedDate(date, "YYYY-MM-DD");
 };
-
+export const formatHumanReadableDateFromDateString = (date?: string | null) => {
+  if (date) {
+    const dateObject = parseISO(date);
+    return format(dateObject, "d MMMM yyyy");
+  }
+  return null;
+};
```

Comment:
What is the case that date params will be null or undefined?

Reply:
ตอนยังไม่ได้เลือกวันอะ

Reply:
กูคิดว่าถ้าใช้ในกรณีเราควรไปกำหนด condition ตอนแรียกใช้แทนไหมนะ

Reply:
โอเค